src/mono/wasm/wasm.proj

```diff
@@ -167,6 +150,31 @@
       <_EmccFlags Include="@(_EmccCommonFlags)" />
     </ItemGroup>
 
+    <PropertyGroup>
+      <_WasmOptConfigurationFlags>@(WasmOptConfigurationFlags->'%(Identity)', '; ') </_WasmOptConfigurationFlags>
```

Comment:
This isn't quite right somehow msbuild still ends up doing a weird thing and I end up with this in the emcc-props.json:

```

  "items": {
    "EmccProperties": [
      { "identity": "RuntimeEmccVersion",     "value": "3.1.7" },
      { "identity": "RuntimeEmccVersionRaw",  "value": "emcc (Emscripten gcc/clang-like replacement + linker emulating GNU ld) 3.1.7 (48a16209b1a0de5efd8112ce6430415730008d18)" },
      { "identity": "RuntimeEmccVersionHash", "value": "48a16209b1a0de5efd8112ce6430415730008d18" }
    ],
    "WasmOptConfigurationFlags": [
      { "identity": "WasmOptConfigurationFlags", "value": "--enable-threads
--enable-bulk-memory
--enable-sign-ext " }
    ]
  }
}
```

note the newlines between the flags.  I have no idea how to get it to just emit semicolons.  (but apparently it works for the cmake flags??

@radical  help

Reply:
Apparently I should use something like `$([MSBuild]::Escape($(MyProp)))`
(seen here: https://docs.microsoft.com/en-us/visualstudio/msbuild/how-to-escape-special-characters-in-msbuild?view=vs-2022)